packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -99,6 +131,35 @@ const FooterTagList = ({post, classes, hideScore, hideAddTag, smallText=false, s
     ssr: false,
   });
 
+  const checkShouldDisplayShowAll = useCallback(() => {
+    if (!showAll && rootRef.current) {
```

Comment:
Just for sakes, should we also check for allowTruncate?

Reply:
Hmm, I think if somehow showAll ends up being false without allowTruncate being set it is better to show the button to expand, so I'm going to leave this as is